ui/src/profile.ts

```diff
@@ -80,9 +80,10 @@ export const fetchFollowing = (): void => {
 export const showNotificationsForFailedProjects = async (): Promise<void> => {
   const failedProjects = await project.fetchFailed();
   failedProjects.forEach(failedProject => {
-    error.show(
-      `The project ${failedProject.metadata.name} could not be loaded`,
-      failedProject
-    );
+    error.show({
+      code: "Project failed to load",
```

Comment:
This doesn't conform to kebab-case :trollface:

Reply:
This was just a test ;) https://github.com/radicle-dev/radicle-upstream/pull/1221/commits/8c334c1ada49d3678869da380952c32af992d743